scripts/run-android-instrumentation-tests.sh

```diff
@@ -1,20 +1,26 @@
 #!/bin/bash
 
+# Python script to run instrumentation tests, copied from https://github.com/circleci/circle-dummy-android
+# Example: ./scripts/run-android-instrumentation-tests.sh com.facebook.react.tests com.facebook.react.tests.ReactPickerTestCase
+#
 export PATH="$ANDROID_HOME/platform-tools:$ANDROID_HOME/tools:$PATH"
 
 # clear the logs
 adb logcat -c
 
 # run tests and check output
-python - $1 << END
+python - $1 $2 << END
+
 import re
 import subprocess as sp
 import sys
 import threading
 import time
 
 done = False
+
 test_app = sys.argv[1]
+test_class = sys.argv[2]
```

Comment:
Idea: make this optional, that way it's still possible to not pass this argument an execute all tests locally (for easy local testing of the whole suite).